src/Datagrid/Pager.php

```diff
@@ -79,4 +82,22 @@ public function init()
             $this->getQuery()->setMaxResults($this->getMaxPerPage());
         }
     }
+
+    /**
+     * Appends a custom tree walker to the tree walkers hint.
+     *
+     * @param AbstractQuery $query
```

Comment:
Please remove this line

Reply:
Why would you want me to delete this line?

Reply:
Because all the information provided by it is already provided by the type hint.